docker-compose.yml

```diff
@@ -338,6 +338,11 @@ services:
     sysctls:
       net.ipv4.ip_unprivileged_port_start: 0
     restart: unless-stopped
+    labels:
+      io.balena.update.strategy: hand-over
+      # We need time for the new JF instance to boot and perform its handover,
+      # so the value here should be higher than the one for the same level at service "api"
+      io.balena.update.handover-timeout: 90000
```

Comment:
@ramirogm Are you trying to ensure that the haproxy instances stick around until the new api instances are up and running? I think if we could ensure that only a single api instance across the entire fleet executed the db setup/migration bits of the backend bootstrap process we'd be in a much better place. What do you think? We already have some kind of "migration" table provided by autumndb I think (don't think its documented very well or at all), I wonder if we could take a lock from that table to ensure a single instance performing db setup/migration and then allowing all other instances to start much faster.

Reply:
Hi @joshbwlng I've tried to explain what issue I'm trying to solve on this [JF improvement](https://jel.ly.fish/user-ramirogm/improvement-enhance-jf-availability-9a7252c@1.0.0#the-new-solution)

Reply:
So, yes that's why I'm trying to achieve around haproxy: stick with the old haproxy because that sends requests to the old-instance but is a kind of side-effect.

I also think that improving the startup of JF could also help here. I'm not sure if the migration is the only issue; there's also concurrency/competition among the workers of the same instance. I need to do a bit of research on this.